AutoRest/Generators/Ruby/Ruby/TemplateModels/MethodTemplateModel.cs

```diff
@@ -517,5 +500,61 @@ private string UrlPathNameFromPathPattern(string urlPathParamName)
             }
             return urlPathParamName;
         }
+
+        /// <summary>
+        /// Constructs mapper for the request body.
+        /// </summary>
+        /// <param name="outputVariable">Name of the output variable.</param>
+        /// <returns>Mapper for the request body as string.</returns>
+        public string ConstructRequestBodyMapper(string outputVariable = "request_mapper")
+        {
+            var builder = new IndentedStringBuilder("  ");
+            if (RequestBody.Type is CompositeType)
+            {
+                builder.AppendLine("{0} = {1}.mapper()", outputVariable, RequestBody.Type.Name);
+            }
+            else
+            {
+                builder.AppendLine("{0} = {{{1}}}", outputVariable,
+                    RequestBody.Type.ConstructMapper(RequestBody.SerializedName, RequestBody, false));
+            }
+            return builder.ToString();
+        }
+
+        /// <summary>
+        /// Creates deserialization logic for the given <paramref name="type"/>.
+        /// </summary>
+        /// <param name="type">Type for which deserialization logic being constructed.</param>
+        /// <param name="valueReference">Reference variable name.</param>
+        /// <param name="responseVariable">Response variable name.</param>
+        /// <returns>Deserialization logic for the given <paramref name="type"/> as string.</returns>
```

Comment:
Added comments and examples at each IType mapper to give and idea of what it'll generate.